Added tests for message function api

DartingMelody opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## General Checkups
- [x] Have you checked that there aren't other open pull requests for the same issue/update/change?
- [x] If your code includes new features and not just bug fixes/copy editing: Did you include new tests?

<!---
by the way: if you want to discuss feature requests and bugs with us before submitting anything on GitHub: We're always happy to chat on Slack: http://slackin.openhumans.org/
 -->

## Description
<!--- Describe your changes in detail -->

## Related Issue
<!--- if this closes an issue make sure include e.g., "fix #4"
or similar - or if just relates to an issue make sure to mention
it like "#4" -->

## Example
<!--- if introducing a new feature or changing behavior of existing
methods/functions, include an example if possible to do in brief form -->

<!--- Did you remember to include tests? Unless you're just changing
grammar, please include new tests for your change -->

madprime commented on ohapi/tests/test_message.py:
```diff
@@ -0,0 +1,64 @@
+                 cassette_library_dir='ohapi/cassettes',
+                 filter_query_parameters=['access_token'])
+
+class test_message(unittest.TestCase):
```
As with #35 I'd like to have this moved to `test_api.py`, and renamed to match PEP8 naming guidelines for classes, e.g. APITestMessage

DartingMelody commented:
Hi @gedankenstuecke @madprime, I have renamed and moved the tests for message function. Please review.

madprime commented on ohapi/tests/test_api.py:
```diff
@@ -158,3 +167,64 @@ def test_get_page_invalid_access_token(self):
+                           subject=SUBJECT, message=MESSAGE,
+                           access_token=MASTER_ACCESS_TOKEN)
+        assert response.json() == {"errors":{"project_member_ids":
+                                  ["Invalid project member ID(s): invalidPMI2"]}}
```
Should try to fix this, e.g. maybe like
```
        assert response.json() == {
            "errors": {"project_member_ids": [
                "Invalid project member ID(s): invalidPMI2"]}}
```

madprime commented on ohapi/tests/test_api.py:
```diff
@@ -158,3 +167,64 @@ def test_get_page_invalid_access_token(self):
+        response = message(project_member_ids=[INVALID_PMI1, INVALID_PMI2],
+                           subject=SUBJECT, message=MESSAGE,
+                           access_token=MASTER_ACCESS_TOKEN)
+        assert response.json() == {"errors":{"project_member_ids":
```
should be `{"errors": {"project_member_ids":`

madprime commented on ohapi/tests/test_api.py:
```diff
@@ -158,3 +167,64 @@ def test_get_page_invalid_access_token(self):
+        response = message(project_member_ids=[INVALID_PMI1, INVALID_PMI2],
+                           subject=SUBJECT, message=MESSAGE,
+                           access_token=MASTER_ACCESS_TOKEN)
+        assert response.json() == {"errors":{"project_member_ids":
```
Should fix this too (whitespace after colon)

madprime commented on ohapi/tests/test_api.py:
```diff
@@ -158,3 +167,64 @@ def test_get_page_invalid_access_token(self):
+                          access_token=ACCESS_TOKEN)
+
+    @my_vcr.use_cassette()
+    def test_message_all_members_false_project_member_id_not_none_invalid_char(self):
```
Is this a test where the project member ID has an invalid character? Maybe `test_message_all_members_false_proj_member_id_has_invalid_char`? (Is that below 79? :) )

madprime commented:
@DartingMelody awesome! No worries on the variables defined in locals(), but there's a couple other style issues still raised by Hound. It seems minor, but we'd like our code to have a standard style, so I'd like those to be fixed. :)

Have you added a linter into your code editor? If not, you might want to look into this. I use linter-flake8 with atom. It's very useful to be able to see issues immediately, while coding!

DartingMelody commented:
Hi, @madprime , thanks for suggesting the linter. I switched to atom with flake8 linter. I have fixed the hound issues(except the locals()). Please review. Thanks! :)

madprime commented:
@DartingMelody Thank you! It's great to have more testing, and I'm glad the tip about a linter was helpful. :) I'll merge now!
